CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ file is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and
 this project adheres to [Semantic
 Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
```

Comment:
We need to add a link at the bottom for this, and I think that will be resolved by rebasing upon the latest main branch.

Reply:
There were merge conflicts with the latest main version in the changelog. What you pointed out has been resolved  during the rebase with main.